packages/errors/src/context.ts

```diff
@@ -582,6 +583,9 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<
         [SOLANA_ERROR__SIGNER__EXPECTED_TRANSACTION_SIGNER]: {
             address: string;
         };
+        [SOLANA_ERROR__SIGNER__NONCE_ACCOUNT_CANNOT_BE_IN_LOOKUP_TABLE]: {
+            signedTransaction: Uint8Array;
```

Comment:
We have to keep in mind that in production, the error context object gets serialized and logged. We must keep sensitive information out of error context, and this probably qualifies as sensitive enough to exclude.